docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md

```diff
@@ -0,0 +1,2506 @@
+
+[Creating cStor storage pools](#creating-cStor-storage-pools)
+
+[Setting Pool Policies](#setting-pool-policies)
+
+[Creating cStor storage classes](#creating-cStor-storage-class)
+
+[Setting Storage Polices](#cstor-storage-policies)
+
+[Monitoring a cStor Pool](#monitor-pool)
+
+[Setting Performance Tunings](#setting-performance-tunings)
+
+[Upgrade the software version of a cStor pool](#Upgrade-the-software-version-of-a-cStor-pool)
+
+[Expanding cStor pool to a new node](#expanding-cStor-pool-to-a-new-node)
+
+[Expanding size of a cStor pool instance on a node by expanding the size of cloud disks](#expanding-size-of-a-cStor-pool-instance-on-a-node-add)
+
+[Expanding size of a cStor pool instance on a node by add physical/virtual disks to a pool instance](#expanding-size-of-a-cStor-pool-instance-on-a-node-add-disk)
+
+[Expanding the cStor Volume Capacity](#expanding-size-of-a-cStor-volume)
+
+[Scaling up cStor Volume Replica](#scaling-up-of-cvr)
+
+[Scaling down cStor Volume Replica](#scaling-down-of-cstor-volume-replica)
+
+## User Operations
+
+### Provisioning a cStor volume {#provisioning-a-cStor-volume}
+
+For provisioning a cStor Volume, it requires a cStor Storage Pool and a StorageClass. The configuration and verification of a cStor Storage pool can be checked from [here](#creating-cStor-storage-pools). The configuration and verification of a StorageClass can be checked from [here](#creating-cStor-storage-class).
+
+Use a similar PVC spec or volumeClaimTemplate to use a StorageClass that is pointing to a pool with real disks. Consider the following parameters while provisioning OpenEBS volumes on real disks.
+
+**AccessModes:** cStor provides iSCSI targets, which are appropriate for RWO (ReadWriteOnce) access mode and is suitable for all types of databases. For webscale applications like WordPress or any for any other NFS needs, you need RWM (ReadWriteMany) access mode. For RWM, you need NFS provisioner to be deployed along with cStor. See how to provision [RWM PVC with OpenEBS](/docs/concepts/rwm)
+
+**Size:** cStor supports thin provisioning by default, which means you can request any size of the volume through the PVC and get it provisioned. Resize of the volume is not fully supported through the OpenEBS control plane in the current release and it is under active development, see [roadmap](/docs/concepts/cstor#cstor-roadmap) for more details. Hence it is recommended to give good amount of buffer to the required size of the volume so that you don't need to resize immediately or in a very short time period. 
+
+The following shows the example PVC configuration for a Deployment and a StatefulSet application which uses a configured StorageClass to provision a cStor Volume. The provided StorageClass should have been configured with StoragePoolClaim property, so the cStor Volume will be provisioned on the StoragePools associated to the StoragePoolClaim.
+
+#### **Example configuration for requesting OpenEBS volume for a Deployment**
```

Comment:
```suggestion
#### Example configuration for requesting OpenEBS volume for a Deployment
```

Heading texts are auto converted to bold

Reply:
Have made the changes- https://github.com/openebs/website/pull/293/commits/6e79209b1999a3142f783cea507f33d65d6ed31f